integration/kubernetes/run_kubernetes_tests.sh

```diff
@@ -95,6 +95,7 @@ else
                 "k8s-shared-volume.bats" \
                 "k8s-volume.bats" \
                 "k8s-nginx-connectivity.bats" \
+		"k8s-block-volume.bats" \
```

Comment:
I think you can remove it from the comment above too ?

Reply:
fixed